testsuite/certificates/__init__.py

```diff
@@ -52,6 +52,20 @@ def __init__(self, binary) -> None:
         super().__init__()
         self.binary = binary
 
+    @staticmethod
+    def build_csr_json(common_name: str,
```

Comment:
Why is this a static method? Couldn't it be a standard function instead?

Reply:
The point was in assigning this builder to the cfssl client class, as this json request is only used by cfssl during certificate creation. But I can move this function to the utils 👍

Reply:
Moved to the standard function above CFSSL class.